fe/app/components/BannerGrid/BannerGrid.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import ImageElement from "../ImageElement/ImageElement";
+import { BannerGridProps } from "./BannerGrid.models";
+
+const BannerGrid = ({ images }: BannerGridProps) => {
+  return (
+    <div>
+      <div className="flex relative overflow-hidden">
+        <div className="flex items-center justify-ceneter min-w-max overflow-hidden gap-3 animate-autoplay-infinite">
+          <BannerGridContent images={images} />
+        </div>
+        <div className="flex items-center justify-end min-w-max overflow-hidden gap-3 animate-autoplay-infinite ml-3">
+          <BannerGridContent images={images} />
+        </div>
+        <div className="flex absolute h-full w-full bg-banner-gradient items-center p-9 md:p-12">
+          <div className="font-league-gothic whitespace-pre-line text-white uppercase text-4xl md:text-7xl">
+            {"Create your own art by\n exchanging ideas with\n other artists"}
```

Comment:
Just here to remind you that after the i18n PR is approve we need to update this texts :)